packages/components/src/higher-order/navigate-regions/index.js

```diff
@@ -54,7 +56,17 @@ export function useNavigateRegions( shortcuts = defaultShortcuts ) {
 			nextRegion = regions[ nextIndex ];
 		}
 
-		nextRegion.focus();
+		// Add the tabindex of -1 so tabbable elements can be located.
+		nextRegion.setAttribute( 'tabindex', '-1' );
+		// Find the next tabbable within the region.
+		const nextTabbable = focus.tabbable.findNext( nextRegion );
+		// Is there a next tabbable available? If not, focus the region instead.
```

Comment:
I'm afraid there's no agreement on this point. Please change to not alter the current behavior.